src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -788,4 +788,26 @@ public static JsonObjectBuilder getChecksumTypeAndValue(DataFile.ChecksumType ch
         }
     }
 
+    public static JsonObjectBuilder json(MapLayerMetadata mapLayerMetadata) {
+        Timestamp lastVerifiedTimestamp = mapLayerMetadata.getLastVerifiedTime();
+        String lastVerifiedString = null;
+        if (lastVerifiedTimestamp != null) {
+            lastVerifiedString = Util.getDateTimeFormat().format(lastVerifiedTimestamp);
+        }
+        return jsonObjectBuilder()
+                .add("id", mapLayerMetadata.getId())
+                .add("embedMapLink", mapLayerMetadata.getEmbedMapLink())
+                .add("joinDescription", mapLayerMetadata.getJoinDescription())
+                .add("layerLink", mapLayerMetadata.getLayerLink())
+                .add("layerName", mapLayerMetadata.getLayerName())
+                .add("mapImageLink", mapLayerMetadata.getMapImageLink())
+                .add("mapLayerLinks", Json.createReader(new StringReader(mapLayerMetadata.getMapLayerLinks())).readObject())
+                .add("worldmapUsername", mapLayerMetadata.getWorldmapUsername())
+                .add("isJoinLayer", mapLayerMetadata.isJoinLayer())
+                .add("dataFileId", mapLayerMetadata.getDataFile().getId())
```

Comment:
Add link to the File page that uses the dataFileId:
```
// example url
https://dataverse.harvard.edu/file.xhtml?fileId={dataFileId}
```

e.g. ```filePageLink``` or current naming standard

```java

String filePageLink = "{domain/server}/file.xhtml?fileId=" + mapLayerMetadata.getDataFile().getId();

// within return jsonObjectBuilder()
             .add("filePageLink", filePageLink)
```